Lib/test/test_enum.py

```diff
@@ -2730,6 +2730,13 @@ def cycle_enum():
         self.assertEqual(256, len(seen), 'too many composite members created')
 
 
+class TestWeirdString(unittest.TestCase):
+    def test_weird_character(self):
+        Yay = Enum('Yay', ('!', 'B', 'C'))
```

Comment:
There should be a test for the original report where empty string causes an error like `Yay = Enum('Yay', ('', 'B', 'C'))` in https://bugs.python.org/issue35899#msg334866

Reply:
Hi @tirkarthi 
Thank you for your feedback, I have just added this test.